resources/lib/utilities.py

```diff
@@ -100,15 +100,6 @@ def findMediaObject(mediaObjectToMatch, listToSearch):
     if result is None and 'ids' in mediaObjectToMatch and 'tvdb' in mediaObjectToMatch['ids'] and mediaObjectToMatch['ids']['tvdb']:
         result = __findInList(
             listToSearch, tvdb=mediaObjectToMatch['ids']['tvdb'])
-    # match by title and year it will result in movies with the same title and
```

Comment:
Better idea, bind this to the setting for name matching fallback?

Reply:
Forgot about this setting but yes, this would make the most sense!

Reply:
Same, totally forgot about it. Saw some discussion on another PR yesterday, which got me thinking we might break some usecase when throwing out match by name completely.

Reply:
Done btw